lib/matplotlib/axes/_base.py

```diff
@@ -1311,7 +1311,7 @@ def set_adjustable(self, adjustable, share=False):
         """
         if adjustable == 'box-forced':
             cbook.warn_deprecated(
-                "2.2", "box-forced", obj_type="keyword argument")
+                "2.2", name="box-forced", obj_type="keyword argument")
```

Comment:
passing obj_type is deprecated and has no effect. Probably doesn't make sense to have a deprecation warning trigger another deprecation warning if we can prevent it...

Reply:
Where is that? I cannot find a deprecation of `obj_type` in the code. Have you maybe misinterpreded the docstring?

~~~
    obj_type : str, optional
        The object type being deprecated.
~~~

Reply:
It was deprecated in #11395 because it does nothing.